baseapp/helpers.go

```diff
@@ -1,10 +1,11 @@
 package baseapp
 
 import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/tendermint/tendermint/abci/server"
 	abci "github.com/tendermint/tendermint/abci/types"
 	cmn "github.com/tendermint/tendermint/libs/common"
+
```

Comment:
Any idea why `goimports` moves some of the imports here, but not others? My understanding is that the imports should be in the format:

```
import (
  std-lib imports

  repo imports

  aliased-repo-imports
)
```

Reply:
Looks like its separating them like:

```
import (
  std-lib imports

  outside-repo-imports

  sdk-imports
)
```

Reply:
No @jackzampolin, it only groups std-libs and non-stdlibs. Within non-stdlibs, you can optionally group libs based on a single project (i.e. cosmos).